[REF] *: General cleanup

luisg123v opened the pull request:
This performs the following changes:
- Update code of the .travis.yml and remove deprecated/unnecessary code
- Fix small typos on the file CONTRIBUTING.md and remove not-useful
  section
- Remove file pseudo_oca_dependencies and replace it by an empty
  oca_dependencies.txt

Closes #1301

luisg123v commented:
@moylop260  
What do you think?

moylop260 commented on .travis.yml:
```diff
@@ -4,46 +4,42 @@ sudo: false
+  postgresql: "9.6"
+  apt:
+    packages:
+      - expect-dev
```
All these packages are not used anymore since that we are using pip wheels with backup of cache directory

luisg123v replied:
@moylop260  
Done. I left apt dependencies as they are, without changes.

luisg123v replied:
@moylop260 
You removed all packages, but I think those were necessary. Travis is failing now (see #1300's log [here](https://travis-ci.org/Vauxoo/addons-vauxoo/jobs/445457431#L958)).

It's failing to run unbuffer because the package  `expect-dev` is not installed.

moylop260 replied:
You are right.
Let’s add that package again

luisg123v committed:
[REF] *: General cleanup

This performs the following changes:
- Update code of the .travis.yml and remove deprecated/unnecessary code
- Fix small typos on the file CONTRIBUTING.md and remove not-useful
  section
- Remove file pseudo_oca_dependencies and replace it by an empty
  oca_dependencies.txt

Closes #1301

moylop260 committed:
[REF] .travis.yml: Remove apt unused packages
